### PR TITLE
feat(timeout): add connect and read timeout default values

### PR DESCRIPTION
**Pull request by raf262**
## Why?
Guzzle expose connect_timeout and read_timeout. In some project we need to use it and set those values.
The bundle weren't exposing the read_timeout. 

## How?
Add node to the configuration :
->floatNode('connect_timeout')->defaultValue(30.0)->end()
->floatNode('read_timeout')->defaultValue(30.0)->end()

### CODE REVIEW

File: src/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -46,6 +46,8 @@ public function getConfigTreeBuilder()
                             ->scalarNode('proxy')->defaultValue('')->end()
                             ->booleanNode('http_errors')->defaultValue(true)->end()
                             ->floatNode('timeout')->defaultValue(5.0)->end()
+                            ->floatNode('connect_timeout')->defaultValue(30.0)->end()
```
Comment (Oliboy50) [inline src/DependencyInjection/Configuration.php:49]: it feels weird to set a `connect_timeout` value to `30` whereas the `timeout` default value is only `5`

something like `1.0` or `2.0` as default should be long enough, no?

**Review by b-viguier — CHANGES_REQUESTED** [review]
Could you also add a test in  `M6WebGuzzleHttpExtension.php` to ensure that this configuration is correctly taken in account? 🙇

File: src/DependencyInjection/Configuration.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (Oliboy50) [reply src/DependencyInjection/Configuration.php:49]: concerning [`read_timeout`](https://docs.guzzlephp.org/en/stable/request-options.html#read-timeout), it seems to only work with `stream: true`, and only used while reading a stream

but is 5 seconds enough to read a stream (by default)? 
I feel like reading a stream could take longer than making a "simple" request

[the remark this answers is not in the document]
Comment (raf262) [reply src/DependencyInjection/Configuration.php]: It depends of the stream size ?

Comment (raf262) [reply src/DependencyInjection/Configuration.php]: I set it to 5.0 both as Benoit suggested me

**Review by b-viguier — APPROVED** [review]
👍 
@Oliboy50 What about a `3.1.0` release? There is no BC 🤷‍♂️

**Review by Oliboy50 — APPROVED** [review]

**Comment by Oliboy50** [comment]
released in `v3.1.0`

**Pull request merged**